doc/neps/ufunc-overrides.rst

```diff
@@ -663,14 +663,14 @@ Symbol Operator     NumPy Ufunc(s)
        (Python 2)
 ``//`` ``floordiv`` :func:`floor_divide`
 ``%``  ``mod``      :func:`mod`
```

Comment:
Is there a reason we use `mod` here and not `remainder`? If `mod` is preferable, then should we reverse the alias, so that `np.mod.__name__ == 'mod'`?